src/ir_Pioneer.cpp

```diff
@@ -30,13 +30,19 @@
 //  http://adrian-kingston.com/IRFormatPioneer.htm
 void IRsend::sendPioneer(const uint64_t data, const uint16_t nbits,
                          const uint16_t repeat) {
-  // If nbits is to big, or is odd, abort.
-  if (nbits > sizeof(data) * 8 || nbits % 2 == 1) return;
-
-  // send 1st part of the code
-  sendNEC(data >> (nbits / 2), nbits / 2, 0);
-  // send 2nd part of the code
-  sendNEC(data & (((uint64_t)1 << (nbits / 2)) - 1), nbits / 2, repeat);
+  for (int32_t r=repeat; r>=0; r--) {
```

Comment:
Please use: `  for (uint16_t r = 0; r <= repeat; r++) {` instead.
a) `int32_t` is a waste of two bytes of stack space. `repeat` is a `uint16_t` ;-)
b) It makes it the same as all the other lines in other protocols. e.g. just a minor style nitpick. :)

Reply:
done